cirq/ops/common_gates.py

```diff
@@ -115,6 +115,41 @@ def _trace_distance_bound_(self) -> Optional[float]:
             return None
         return abs(np.sin(self._exponent * 0.5 * np.pi))
 
+    def controlled(self,
+                   num_controls: int = None,
+                   control_values: Optional[Sequence[
+                       Union[int, Collection[int]]]] = None,
+                   control_qid_shape: Optional[Tuple[int, ...]] = None
+                  ) -> raw_types.Gate:
+        """
+        Constructs CNotPowGate from controlled XPowGate when applicable.
+
+        This method is a specialized controlled method for XPowGate. It
+        overrides the default behavior of returning a ControlledGate by
+        transforming the underlying controlled gate to a CNotGatePow and
+        removing the last specified control qubit (which acts first
+        semantically).  If this is a gate with multiple control qubits, it will
+        now be a ControlledGate with one less control.
+
+        This behavior only occurs when the last control qubit is a default-type
+        control qubit. A default-type control qubit is one with shape of 2 (not
+        a generic qudit) and where the control is satisfied by the qubit being
+        ON, as opposed to OFF.
+
+        (Note that a CNotPowGate is, by definition, a controlled-XPow gate.)
```

Comment:
Optional:  Would controlled-XPowGate be better?  I'll leave it up to you.  Same comments below.

Reply:
Looks better. Made the changes.